src/cli.ts

```diff
@@ -226,6 +235,10 @@ export function run (process: NodeJS.Process): void {
     myConsole
   ).buildFromProjectDir(projectDir, process)
 
+  if (options.addLicenseText) {
```

Comment:
Why do this after everything is set? 
I'd suggest putting the code in the existing `BomBuilder.makeComponent()`